event-log/src/main/scala/io/renku/eventlog/tsMigrationModel.scala

```diff
@@ -16,37 +16,19 @@
  * limitations under the License.
  */
 
-package io.renku.eventlog.subscriptions.tsmigrationrequest
+package io.renku.eventlog
 
-import cats.Show
 import cats.syntax.all._
 import io.circe.Decoder
 import io.renku.data.ErrorMessage
-import io.renku.eventlog.subscriptions.{Capacity, SubscriptionInfo}
-import io.renku.events.consumers.subscriptions.{SubscriberId, SubscriberUrl}
-import io.renku.http.server.version.ServiceVersion
 import io.renku.tinytypes.constraints.{InstantNotInTheFuture, NonBlank}
 import io.renku.tinytypes.{InstantTinyType, StringTinyType, TinyTypeFactory}
 
 import java.time.Instant
 
-private final case class MigratorSubscriptionInfo(subscriberUrl:     SubscriberUrl,
-                                                  subscriberId:      SubscriberId,
-                                                  subscriberVersion: ServiceVersion
-) extends SubscriptionInfo {
-  override val maybeCapacity: Option[Capacity] = None
-}
-
-private object MigratorSubscriptionInfo {
-  implicit lazy val show: Show[MigratorSubscriptionInfo] = Show.show { info =>
-    import info._
-    show"subscriber = $subscriberUrl, id = $subscriberId, version = $subscriberVersion"
-  }
-}
-
-private sealed trait MigrationStatus extends StringTinyType with Product with Serializable
+sealed trait MigrationStatus extends StringTinyType with Product with Serializable
```

Comment:
Why is this public now?

Reply:
It's because it's shared between `TS_MIGRATION_REQUEST` and `MIGRATION_STATUS_CHANGE` categories.